src/main.ts

```diff
@@ -108,6 +109,13 @@ async function runWorkflow(bufPath: string, inputs: Inputs): Promise<Steps> {
   if (checks.some((result) => result.status == Status.Failed)) {
     return steps;
   }
+  const modules = parseModules(inputs.input);
+  if (modules.length == 0) {
+    core.info("No modules found, skipping push and archive");
```

Comment:
there might be modules, they just don't have names to push to the BSR

Reply:
Fixed to `module names`